pgrx/src/datum/geo.rs

```diff
@@ -59,11 +150,11 @@ impl FromDatum for pg_sys::Point {
     }
 }
 
-impl IntoDatum for pg_sys::Point {
+impl IntoDatum for Point {
     fn into_datum(mut self) -> Option<pg_sys::Datum> {
         unsafe {
             let copy = PgMemoryContexts::CurrentMemoryContext
-                .copy_ptr_into(&mut self, std::mem::size_of::<pg_sys::Point>());
+                .copy_ptr_into(&mut self, std::mem::size_of::<Point>());
```

Comment:
we tend to prefix things in `pg_sys` with `pg_sys::`.  I can't say we're 100% consistent with that throughout the codebase, but my preference would be to shift towards that direction, not away.

The reason is it makes it a lot easier when reading the code to know "this thing is defined by Postgres", which is almost always an important thing to know about a type when working inside pgrx.

So, if you will, please revert this particular change, and if there's anything else in the PR that elides `pg_sys::`, please add it.

Reply:
To be clear, this isn't quite "eliding `pg_sys::`. This is a change from `pg_sys::Point` to `geo::Point` (which happens to be an alias to `pg_sys::Point`). It's especially confusing here because they are both spelled and capitalized the same way, but other cases it's `Box` vs `pg_sys::BOX` and `LineSegment` vs `pg_sys::LSEG`.

Also, this particular line could also be rewritten as `std::mem::size_of::<Self>`.

I do like the more normal type names, especially in place of ALL_CAPS ones, but if `pg_sys::` is preferred I can make that change.

Reply:
> This is a change from pg_sys::Point to geo::Point

hmm.  yeah, it is.

> Also, this particular line could also be rewritten as std::mem::size_of::<Self>.

Indeed.  Probably good to make that little change.

> but if pg_sys:: is preferred I can make that change.

No, this is correct.  You're right.

Reply:
Updated.